docs/source/installation.mdx

```diff
@@ -0,0 +1,41 @@
+<!---
+Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Installation
+
+To install the latest release of 🤗 Optimum Intel with the corresponding required dependencies, you can do respectively:
+
+| Accelerator                                                                                                            | Installation                                        |
+|:-----------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------|
+| [Intel Neural Compressor (INC)](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html) | `python -m pip install "optimum[neural-compressor]"`|
+| [Intel OpenVINO](https://docs.openvino.ai/latest/index.html)                                                           | `python -m pip install "optimum[openvino,nncf]"`    |
```

Comment:
It would be useful to mention that to do inference on stable diffusion models, you need to do `pip install optimum-intel[diffusers]` or a note to manually install diffusers if  we do not need a specific version

Reply:
I will add it in the documentation under the "Stable Diffusion" section